script.js

```diff
@@ -2,15 +2,264 @@ const mobileMenu = document.querySelector('#mobile-menu-modal');
+    description: cardModel1Informations[1],
+    techsList: cardModel1Informations[2],
+    project: cardModel1Informations[3],
+  },
+  workModel2: {
+    title: cardModel2Informations[0],
+    description: cardModel2Informations[1],
+    techsList: cardModel2Informations[2],
+    project: cardModel2Informations[3],
+  },
+  item2: {
+    title: cardItem2Title,
+  },
+  item3: {
+    title: cardItem3Title,
+  },
+  cardModel: {
+    classes: [
+      'work_model_1',
+      'work_model_2 item-1',
+      'work_model_2 item-2',
+      'work_model_2 item-3',
+      'work_model_2',
+      'work_model_2 item-2',
+      'work_model_2 item-3',
+    ],
+  },
+};
 
 function toggleMobileMenu() {
   mobileMenu.classList.toggle('toogleModalDisplay');
 }
 
+const addCardTitle = (modelData) => {
+  const title = document.createElement('h2');
+  title.textContent = projectsData[modelData].title;
+  return title;
+};
+const addCardDescription = (modelData) => {
+  const description = document.createElement('p');
+  description.textContent = projectsData[modelData].description;
+  return description;
+};
+const addTechsList = (modelData) => {
+  const modelTechsList = document.createElement('ul');
+  for (let i = 0; i < projectsData[modelData].techsList.length; i += 1) {
+    const listElement = document.createElement('li');
+    modelTechsList.appendChild(listElement);
+    const listButton = document.createElement('button');
+    listButton.textContent = projectsData[modelData].techsList[i];
+    listElement.appendChild(listButton);
+  }
+
+  return modelTechsList;
+};
+const addProjectButton = (modelData) => {
+  const projectButton = document.createElement('button');
+  projectButton.className = 'orange_button';
+  projectButton.textContent = projectsData[modelData].project;
+  return projectButton;
+};
+
+const createfirstCard = (card) => {
+  const modelData = 'workModel1';
+  const modelPicture = document.createElement('div');
+  modelPicture.className = 'model_1-picture';
+  card.appendChild(modelPicture);
+  const modelContent = document.createElement('div');
+  modelContent.className = 'model_1-content';
+  card.appendChild(modelContent);
+  const modelTitle = document.createElement('div');
+  modelTitle.className = 'model_1-title';
+  modelContent.appendChild(modelTitle);
+  modelTitle.appendChild(addCardTitle(modelData));
+  const modelDescription = document.createElement('div');
+  modelDescription.className = 'model_1-description';
+  modelContent.appendChild(modelDescription);
+  modelDescription.appendChild(addCardDescription(modelData));
+  const modelTechs = document.createElement('div');
+  modelTechs.className = 'model_1-tech';
+  modelContent.appendChild(modelTechs);
+  modelTechs.appendChild(addTechsList(modelData));
+  const modelProject = document.createElement('div');
+  modelProject.className = 'model_1-project';
+  modelContent.appendChild(modelProject);
+  modelProject.appendChild(addProjectButton(modelData));
+
+  return card;
+};
```

Comment:
[OPTIONAL] I noticed for dynamically creating the project cards, you relied on the `document.createElement()` method, which is completely fine. You can consider using `Template literals` to create them instead, as it will make your code more legible, clearer and cleaner. You can refer to this [link](https://wesbos.com/template-strings-html) for more information about the topic.